eth/stagedsync/stage_execute.go

```diff
@@ -155,13 +156,15 @@ func executeBlock(
 		return h
 	}
 
-	getTracer := func(txIndex int, txHash common.Hash) (vm.EVMLogger, error) {
-		return tracelogger.NewStructLogger(&tracelogger.LogConfig{}), nil
+	getTracer := func(txIndex int, txHash common.Hash) (*tracing.Hooks, error) {
+		return tracelogger.NewStructLogger(&tracelogger.LogConfig{}).Hooks(), nil
 	}
 
 	callTracer := calltracer.NewCallTracer()
 	vmConfig.Debug = true
-	vmConfig.Tracer = callTracer
+	if vmConfig.Tracer == nil {
```

Comment:
plz add it into into `cmd/state/exec3/state.go:RunTxTaskNoLock`

Reply:
It's already handled there via https://github.com/ledgerwatch/erigon/blob/161736348c38281afc9d670a89cc9c6097be8b72/cmd/state/exec3/state.go#L82